misc/test/bitcoin.conf

```diff
@@ -0,0 +1,9 @@
+regtest=1
+fallbackfee=0.0004
+txindex=1
+server=1
+rpcuser=bitcoin
+rpcpassword=bitcoin
+zmqpubrawblock=tcp://127.0.0.1:38332
+# commonly used port 28333 collides with dkg ports
+zmqpubrawtx=tcp://127.0.0.1:38333
```

Comment:
Will this conflict with #1851? Might want to be clever about merge order.

Reply:
I think we can merge whichever is approved first, and I can fix conflicts if there are any.